src/main/java/com/scrive/pdftools/tools/utils/S3ForLambda.java

```diff
@@ -0,0 +1,94 @@
+package com.scrive.pdftools.tools.utils;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.*;
+
+import java.io.*;
+import java.util.Optional;
+
+public class S3ForLambda {
+
+    private static final String ENV_PARAM_ACCESS_KEY = "access_key";
+    private static final String ENV_PARAM_SECRET_KEY = "secret_key";
+    private static final String ENV_PARAM_BUCKET = "bucket";
+    private static final String ENV_PARAM_FAKES3_HOST = "fakes3_host";
+    private static final String ENV_PARAM_FAKES3_PORT = "fakes3_port";
+    private static final Regions S3_REGION = Regions.EU_WEST_1;
+
+
+    public static String fromStreamToString(InputStream inputStream) throws IOException {
+        final int bufferSize = 1024;
+        final char[] buffer = new char[bufferSize];
+        final StringBuilder out = new StringBuilder();
+        Reader in = new InputStreamReader(inputStream, "UTF-8");
+        for (; ; ) {
+            int rsz = in.read(buffer, 0, buffer.length);
+            if (rsz < 0)
+                break;
+            out.append(buffer, 0, rsz);
+        }
+        return out.toString();
+    }
+
+    private static ByteArrayOutputStream fromStreamToByteArrayOutputStream(InputStream inputStream) throws IOException {
+        final int bufferSize = 1024;
+        final byte[] buffer = new byte[bufferSize];
+        final ByteArrayOutputStream out = new ByteArrayOutputStream();
+        for (; ; ) {
+            int rsz = inputStream.read(buffer, 0, buffer.length);
```

Comment:
at the very least `IOUtils.copy(inputStream, out);` but maybe we can get rid of this method altogether

Reply:
if we get rid of  getBytesFromAmazonFile since it is unused, we can remove this one as well

Reply:
Yes. Removed

Reply:
Yes. This is removed